### PR TITLE
Picking Terminal 1: Fix filtering HUs to pack view

### PR DESCRIPTION
**Pull request by teosarca**

### CODE REVIEW

File: backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/pickingslotsClearing/PackingHUsViewFactory.java [hunk 1]
```diff
@@ -222,4 +233,11 @@ private IHUQueryBuilder createHUQuery(final PackingHUsViewKey key)
 		}
 		return huQuery;
 	}
+
+	private void onViewRemoved(final RemovalNotification<Object, Object> notification)
+	{
+		final IView view = (IView)notification.getValue();
+		logger.debug("View <" + view.getViewId() + "> removed from cache. Cause: " + notification.getCause());
```
Comment (teosarca) [inline backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/pickingslotsClearing/PackingHUsViewFactory.java:240]: avoid string concatenation

File: backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/pickingslotsClearing/PackingHUsViewFactory.java [hunk 2]
```diff
@@ -108,15 +120,15 @@ public WindowId getWindowId()
...
+	public void setViewsRepository(final IViewsRepository viewsRepository)
 	{
 		this.viewsRepo = viewsRepository;
 	}
 
 	@Override
 	public void put(final IView view)
 	{
-		throw new UnsupportedOperationException();
+		views.put(view.getViewId(), view);
```
Comment (teosarca) [inline backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/pickingslotsClearing/PackingHUsViewFactory.java:131]: not sure this OK.
we are storing the PackingHUsView inside the PickingSlotsClearingView

Comment (adi-stefan) [reply backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/pickingslotsClearing/PackingHUsViewFactory.java]: removed.

**Comment by adi-stefan** [comment]
The filter call no longer fails, but there's still an error in when calling the `quickActions` endpoint for the child view
image

**Pull request merged**